observability/monitoring/getting-started/maintenance-and-support-for-monitoring.adoc

```diff
@@ -6,7 +6,23 @@ include::_attributes/common-attributes.adoc[]
 
 toc::[]
 
-TBD
+Not all configuration options for the monitoring stack are exposed. The only supported way of configuring {product-title} monitoring is by configuring the {cmo-first} using the options described in the xref:../../../observability/monitoring/config-map-reference-for-the-cluster-monitoring-operator.adoc#cluster-monitoring-operator-configuration-reference[Config map reference for the {cmo-full}]. *Do not use other configurations, as they are unsupported.*
```

Comment:
In general, [repo guidelines](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#quick-markup-reference) mention avoiding markup for emphasis, but shows using italics if it must be.

Maybe an `[IMPORTANT]` admonition box or similar would be useful? Or just dropping the markup would probably be fine, too.